builtin/providers/azurerm/resource_arm_loadbalancer_probe.go

```diff
@@ -128,7 +128,18 @@ func resourceArmLoadBalancerProbeCreate(d *schema.ResourceData, meta interface{}
 		return fmt.Errorf("Cannot read LoadBalancer %s (resource group %s) ID", loadBalancerName, resGroup)
 	}
 
-	d.SetId(*read.ID)
+	var createdProbe_id string
```

Comment:
We could use retrieveLoadBalancerById() here again then findLoadBalancerProbeByName(), but it's another call to the API.